.github/workflows/build-docs.yml

```diff
@@ -258,6 +258,11 @@ jobs:
            nilearn_data/spm_multimodal_fmri
           key: v1-spm_multimodal_fmri-${{ hashFiles('week_num') }}
 
+      # Update the authors file and the names file
+      # in case a contributor has been added to citation.cff
+      # but did not run the maint_tools/citation_cff_maint.py script.
+      - name: update AUTHORS.rst and doc/changes/names.rst
+        run: python maint_tools/citation_cff_maint.py
```

Comment:
Since running `citation_cff_maint.py` is automated can we keep the instructions simple for 1st time contributors so all they have to do is add their names to `citation.cff` ?

Reply:
yup I was thinking about that too: will update

Reply:
update the contributing.rst also with an example of what to add in the citation.cff